src/Survey.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Survey helper methods.
+ */
+
+namespace Automattic\WooCommerce\Admin;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Survey Class.
+ */
+class Survey {
+	/**
+	 * Get a survey's URL from a path.
+	 *
+	 * @param  string $path Path of the survey.
+	 * @param  array  $query Query arguments as key value pairs.
+	 * @return string Full URL to survey.
+	 */
+	public static function get_url( $path, $query = array() ) {
+		$url = 'https://automattic.survey.fm' . $path;
```

Comment:
Would it be worth to making the `automattic.survey.fm` url a class constant:
`const SURVEY_URL = 'https://automattic.survey.fm'`

Reply:
Seems like a reasonable idea.  Added in https://github.com/woocommerce/woocommerce-admin/pull/6066/commits/9b97321da6fe649e9c4909d5d7614f1b9ea551e6